deps/v8/include/v8-version.h

```diff
@@ -17,4 +17,7 @@
 // (Boolean macro values are not supported by all preprocessors.)
 #define V8_IS_CANDIDATE_VERSION 0
 
+// Modify this when changes are made by the embedder.
+#define V8_EMBEDDER_VERSION "node.0"
```

Comment:
I like the idea but I'm not sure how to update the other files. Should I add conditions everywhere to check if the embedder string is empty ?